### PR TITLE
Added new parameter dnssleep

### PR DESCRIPTION
**Pull request by rajcz**
Added new parametr dnssleep for better handling sleep time after succesful insert of records via dns api

### CODE REVIEW

**Comment by Neilpang** [comment]
It's not enough.  It's also needed to save such sleep time to the domain conf file, so that the auto-renewal can get use it.

**Comment by rajcz** [comment]
OK, this I fix 1) save to conf 2) default 60s sleep. But If I understand right - I must add parametr to issue function with sleep too? Or can you help me with renew in code, thanks.

**Comment by rajcz** [comment]
Hi Neilpang, today I completed all for dnssleep parametr. Tested and all is ok, from renew action too. Pavel

File: acme.sh [hunk 1]
```diff
@@ -1105,7 +1105,8 @@ issue() {
   Le_RealCACertPath="$7"
   Le_ReloadCmd="$8"
   Le_RealFullChainPath="$9"
-  
+  dnssleep="${10}"
```
Comment (Neilpang) [inline acme.sh]: why not name it as something like:  "Le_Dns_Sleep"  ?

File: acme.sh [hunk 2]
```diff
@@ -2235,6 +2236,7 @@ _process() {
   _accountkey=""
   _certhome=""
   _httpport=""
+  dnssleep="60"
```
Comment (Neilpang) [inline acme.sh]: why not  _dnssleep  ?

File: acme.sh [hunk 3]
```diff
@@ -2361,13 +2363,13 @@ _process() {
           _webroot="$_webroot,$wvalue"
         fi
         ;;
-    --dnssleep 
+    --dnssleep)          
         re='^[0-9]+$'
-        if ! [[ $2 =~ $re ]] ; then
```
Comment (Neilpang) [inline acme.sh]: Do not use "[[",  it's bash only.   We need keep sh compatible.
Use "["  instead.

File: acme.sh [hunk 4]
```diff
@@ -2361,13 +2363,13 @@ _process() {
...
-           dnssleep="60"
+        if ! [[ "$2" =~ $re ]] ; then
```
Comment (Neilpang) [inline acme.sh]: ??

**Comment by Neilpang** [comment]
fixed  0e38c60da35620059ca3aa99a0982276260fc3c4

So, close.

**Pull request closed without merging**